### PR TITLE
BaSP-TG-108: Update Project Form

### PR DESCRIPTION
**Pull request by Gearyandres94**

### CODE REVIEW

File: src/components/Project/index.js [hunk 1]
```diff
@@ -171,114 +165,166 @@ function ProjectForm() {
...
-                  <EmployeeItem key={index} employee={employee} />
-                ))}
-              </tbody>
-            </table>
+              )}
+            />
+            <Controller
+              control={control}
+              name="description"
+              render={({ field: { value, onChange }, fieldState: { error } }) => (
```
Comment (laura-brussa12) [inline src/components/Project/index.js:215]: Same as in the above

File: src/components/Project/index.js [hunk 2]
```diff
@@ -171,114 +165,166 @@ function ProjectForm() {
...
           </div>
+          <div className={styles.addEmployeeDiv}>
+            <ul className={styles.employeeUl}>
+              {fields.map((field, index) => (
+                <div key={field.id} className={styles.employeeDiv}>
+                  <Controller
+                    control={control}
+                    name={`employees[${index}].employeeId`}
+                    defaultValue={field.text}
+                    render={({ field: { value, onChange }, fieldState: { error } }) => (
```
Comment (laura-brussa12) [inline src/components/Project/index.js:270]: Same as in the above

File: src/components/Project/index.js [hunk 3]
```diff
@@ -171,114 +165,166 @@ function ProjectForm() {
...
+                        placeholder="Rate"
+                        onChange={onChange}
+                        error={error?.message}
+                      />
+                    )}
+                  />
+                  <Controller
+                    control={control}
+                    name={`employees[${index}].role`}
+                    render={({ field: { value, onChange }, fieldState: { error } }) => (
```
Comment (laura-brussa12) [inline src/components/Project/index.js:299]: Same as in the above

File: src/components/Project/index.js [hunk 4]
```diff
@@ -1,161 +1,155 @@
...
-        openModal();
-      } else {
-        dispatch(postProject(body));
-        setModalTitle('Project created');
-        setMsg('Project created successfully!');
-        openModal();
-      }
+  }, [errorDB]);
+  const onSubmit = (data) => {
+    console.log(data, 'data');
```
Comment (AlejandroSantini) [inline src/components/Project/index.js:135]: delete this console.log

**Review by GuidoCerioni — APPROVED** [review]
Nice work Andy

**Review by HernanCazeneuve — DISMISSED** [review]

**Review by GuidoCerioni — APPROVED** [review]

**Review by CarVigil — APPROVED** [review]

**Review by HiginiaMed — APPROVED** [review]

**Pull request merged**